docs/community/team.md

```diff
@@ -56,21 +56,64 @@ Early development on the project was led by Kira Evans, first as an intern with
-
-## Institutional and funding Partners
-
-- [Chan Zuckerberg Initiative Foundation.](https://chanzuckerberg.com/science) Representative [Kyle I. S. Harrington](https://github.com/kephale).
-
-    - The napari project and community have been generously supported by the [CZI Science Imaging Program.](https://chanzuckerberg.com/science/programs-resources/imaging/)
-    - Since late 2018, Juan Nunez-Iglesias has been supported by a CZI Imaging Software Fellowship.
-    - The CZI Imaging Tech Team, under the leadership of Nicholas Sofroniew and engineering management of Justine Larsen, has been making direct engineering and design contributions to improve and expand napari.
-    - The CZI Imaging Tech Team has supported [Quansight Labs](https://labs.quansight.org) and independent contractors to improve and expand napari.
-    - The CZI Imaging Tech Team, through an effort led by Justin Kiggins, has created [napari-hub.org](https://www.napari-hub.org/), a site to make discovering and sharing napari plugins easier.
-    - CZI has launched a [grant program](https://chanzuckerberg.com/rfa/napari-plugin-grants/) to help accelerate the napari plugin developer community.
-
-The current representative of the Institutional and Funding Partner Advisory Council on the napari steering council is Kyle I. S. Harrington.
+Over time, napari has grown to over 180 direct contributors. Talley Lambert,
+from Harvard Medical School, began contributing around the time of Juan's blog
+post, and has been a critical driver of napari's development, including
+development of its first and second (current) plugin systems.
+
+We have now have core team members located in four continents and three
+fortnightly community calls spread across the day to accommodate users
+and contributors in any time zone.
+
+Read more about napari’s [mission and values](mission-and-values), how to get
+started as a [contributor](napari-contributing) or join us in our
+[public chat room](https://napari.zulipchat.com/login/). You can also follow us
+on [mastodon](https://fosstodon.org/@napari),
+[Bluesky](http://bsky.app/profile/napari.org), or
+[LinkedIn](https://www.linkedin.com/company/napari).
+
+(funding)=
+## Institutional and funding partners
+
+napari is developed by a global community of scientists, with many working as
+volunteers, improving napari for their own use while sharing their work with
+the world. We appreciate their efforts!
+
+We have also received generous direct funding and in-kind support that
+enables us to develop faster:
+
+- The [Chan Zuckerberg Initiative] has long supported napari in numerous ways
+  (listed in reverse chronological order):
+
+  - CZI is currently supporting napari with [two grants][sustainability]
```

Comment:
@jni I know you don't like the name "scaffolding grant" but I really think it should be a separate line item that also lists the amount (since we don't have a grant page for that one)